includes/discovery/sensors/temperature/loop-telecom.inc.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
Hi,
You can probably do it in YAML and provide snmp_hexstring as user-func.

Reply:
Could you provide me with an example on how to do this? 
I could not find this anywhere in the documentation. 

Thank you!